pom.xml

```diff
@@ -56,7 +56,7 @@
 			<dependency>
 				<groupId>com.google.guava</groupId>
 				<artifactId>guava</artifactId>
-				<version>27.0.1-jre</version>
+				<version>30.0-jre</version>
```

Comment:
Why not [30.1.1-jre](https://search.maven.org/artifact/com.google.guava/guava/30.1.1-jre/bundle)?

Reply:
I went with the minimum version that listed the vulnerability as fixed by CRDA. I'll test 30.1.1 to see if it works, and use it instead if it does.